packages/crypto-wallet-core/src/transactions/eth/index.ts

```diff
@@ -8,22 +8,44 @@ export class ETHTxProvider {
     gasPrice: number;
     data: string;
     gasLimit: number;
+    network: string;
     chainId?: number;
   }) {
-    const { recipients, nonce, gasPrice, data, gasLimit, chainId = 1} = params;
+    const { recipients, nonce, gasPrice, data, gasLimit, network, chainId } = params;
     const { address, amount } = recipients[0];
+    const chainCode = this.getChainCode(network, chainId);
     const txData = {
       nonce: utils.toHex(nonce),
       gasLimit: utils.toHex(gasLimit),
       gasPrice: utils.toHex(gasPrice),
       to: address,
       data,
       value: utils.toHex(amount),
-      chainId
+      chainId: chainCode
```

Comment:
please unify the name chainId / chainCode.

Reply:
unified